controllers/profiles/object_creators.go

```diff
@@ -59,11 +58,13 @@ const (
 	healthStartedInitialDelaySeconds = 10
 )
 
-var defaultDevApplicationProperties = "quarkus.http.port=" + strconv.Itoa(defaultHTTPWorkflowPort) + "\n" +
+var defaultDevApplicationProperties = "quarkus.http.port=8080\n" +
 	"quarkus.http.host=0.0.0.0\n" +
 	// We disable the Knative health checks to not block the dev pod to run if Knative objects are not available
 	// See: https://kiegroup.github.io/kogito-docs/serverlessworkflow/latest/eventing/consume-produce-events-with-knative-eventing.html#ref-knative-eventing-add-on-source-configuration
-	"org.kie.kogito.addons.knative.health-enabled=false\n"
+	"org.kie.kogito.addons.knative.health-enabled=false\n" +
```

Comment:
I don't think it will work.
These are quarkus properties expected to be set here.

If we need to set envs, we should add to the Deployment instead.
As Quarkus accepts envs and system properties, in this case here, it is better to use the correspondent system property.

I don't remember the wight of the envs and sys props on quarkus smallrye config, but IIRC, system properties has preference over envs.
It might worth to check though.

Reply:
Oh, you updated it just while I was writing this comment :)

Reply:
@spolti changed with the properties